extras/printschema.py

```diff
@@ -0,0 +1,45 @@
+import copy
```

Comment:
wrt the idea: Thanks for explaining your use-case to me. It's great to hear how people are using this library!

I don't know that this belongs in the library itself though, but I'd love to support that functionality in a flexible way where you don't have to literally hack the library to make it work. Maybe we could add a hook to enable custom renderers, something like the way it works in the [`json` library](https://docs.python.org/3/library/json.html#json.JSONEncoder), but hopefully a little simpler.

Still I don't anticipate this making version `0.2`. I really want to get those bugfixes out there ASAP, so we'll wait until the next version for this.